draft-ietf-rats-architecture.md

```diff
@@ -249,13 +179,7 @@ The system components perform a series of measurements that may be
 signed by the root of trust, considered as Evidence about the hardware,
```

Comment:
This PR is problematic in that it starts using capitalized terms like Evidence before being defined

Reply:
Evidence is briefly defined in the introduction as well as more formally covered in the Terminology section (also Attester and Relying Party).